### PR TITLE
package + readme cleanups

### PR DESCRIPTION
**Pull request by alkasm**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,17 +54,27 @@ run = nm.get_run('ri.scout.gov-staging.run.ce205f7e-9ef1-4a8b-92ae-11edc77441c6'
...
+By default, the library uses `https://api.gov.nominal.io/api` as the base url to the Nominal platform. Your scripts can change the URL they use with `set_base_url()`. For example, to use the staging URL:
+
+```py
+nm.set_base_url('https://api-staging.gov.nominal.io/api')
+```
+
 ## Development
 
-Developer workflows are ran with [`just`](https://github.com/casey/just). You can use `just -l` to list commands, and view the `justfile` for the commands (they are all very simple, and should remain that way).
+Developer workflows are ran with [`just`](https://github.com/casey/just). You can use `just -l` to list commands, and view the `justfile` for the commands.
```
Comment (alxhill) [inline README.md:67]:
```suggestion
Developer workflows are run with [`just`](https://github.com/casey/just). You can use `just -l` to list commands, and view the `justfile` for the commands.
```

File: pyproject.toml [hunk 2]
```diff
@@ -2,33 +2,31 @@
...
-authors = ["JP <jack@nominal.io>", "Michael Bauer <michael@nominal.io>", "Alexander Reynolds <alex.reynolds@nominal.io>"]
-readme = "README.md"
-packages = [
-    {include = "nominal/py.typed"},
+authors = [
+    "JP <jack@nominal.io>",
+    "Michael Bauer <michael@nominal.io>",
+    "Alexander Reynolds <alex.reynolds@nominal.io>",
 ]
+maintainers = ["Alexander Reynolds <alex.reynolds@nominal.io>"]
```
Comment (alxhill) [inline pyproject.toml:10]: all alone? 🥺

File: pyproject.toml [hunk 3]
```diff
@@ -2,33 +2,31 @@
...
 conjure-python-client = ">=2.8.0,<3"
 pandas = ">=2.2,<3.0"
 pydantic = "^2.9.1"
 typing-extensions = "^4.12.2"
 click = "^8.1.7"
 
-[tool.poetry.dev-dependencies]
-isort = "^5.13.2"
-ruff = "^0.4.2"
-
```
Comment (alxhill) [inline pyproject.toml:29]: cool I made these changes too so we should centralise on this PR - I'll push the change in a sec

**Review by alxhill — APPROVED** [review]

File: pyproject.toml (hunk 2 above)
Comment (alkasm) [reply pyproject.toml:11]: add yourself! :)

**Pull request merged**